app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -193,47 +214,108 @@ You will not be eligible for the IRP, because you have spent more than 90 days i
 
-* teaching in Scotland, Wales or Northern Ireland
 
 ## How to apply for the international relocation payment (IRP)
 
-Applications for the international relocation payment (IRP) are now closed. You can express your interest in the IRP by emailing us at IRP.ExpressInterest@education.gov.uk. We'll email you when applications reopen in autumn 2024.
+We will update this page with a link to the application form when it is live.
+
+Before you apply for the first and second instalments of the IRP, you’ll need to check you meet the eligibility requirements.
+
+Applications are open from 30 September 2024 to end of March 2025
+
+If you start your teaching job between 1 March 2024 and 28 February 2025 you should apply between 30 September 2024 and the end of March 2025. If you are eligible, you should receive the money following successful completion of our eligibility checks.You must apply between 30 September 2024 and the end of March 2025 to remain eligible.
+
+
+## DfE eligibility requirements for IRP instalments
+
+### First instalment of IRP
+
+DfE will check you meet the eligibility requirements for the first instalment of IRP by validating your application with:
+
+* [UK Visas and Immigration](https://www.gov.uk/government/organisations/uk-visas-and-immigration) 
 
-You'll need to have started your teaching job to apply for the IRP. [Learn more about finding a job and applying for your visa](/non-uk-teachers/teach-in-england-if-you-trained-overseas). 
+* the school where you are employed.
 
-Eligible applicants who apply for the IRP after 1 September 2024 will receive two £5,000 payments over 2 years, as long as they remain eligible. Teachers will be able to apply for the first payment at the start of their first year teaching in England and the second payment at the start of their second year. We’ll publish further details about how these split payments will be made to teachers on this page in due course.
+If your eligibility checks are successful, you will get the first instalment of £5,000. 
 
+### Second instalment of IRP
 
+At the start of your second year of employment as a teacher, apply for the second instalment of the IRP. To get the second instalment of the IRP, you will need to have:
 
-## General information about the international relocation payment
+* made a successful claim for the first instalment
+
+* completed 3 consecutive terms of employment as a teacher
+
+Applications for the second instalment are open from September 2025 to March 2026.
+
+The Department for Education will validate your application with [UK Visas and Immigration](https://www.gov.uk/government/organisations/uk-visas-and-immigration) and the school or schools where you have worked since applying for the first instalment of the IRP. If you meet the eligibility requirements for the second instalment set out on this page, you should receive the second instalment of £5,000 following successful completion of our eligibility checks. 
```

Comment:
Again, do we need this link? If we do need this link on this page, we should only include it once for accessibility reasons